CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## v0.9.0 (TBD)
 
-
+- Adapt to the use of `TransactionHeader` in batches and blocks (#781).
```

Comment:
Our changelog is intended for the end users / node operator so I don't think this PR is "visible" to them.

```suggestion
```

Reply:
Makes sense, I removed it. I can't seem to add labels to this PR, so maybe the no changelog label has to be added still.